helm/charts/determined/Chart.yaml

```diff
@@ -1,12 +1,19 @@
 apiVersion: v1
 name: determined
 description: A Helm chart for Determined
-version: "0.37.1-dev0"
+
+# This is a dummy value. The actual version string is passed through the
```

Comment:
im a dummy and i find this offensive
https://docs.splunk.com/Documentation/StyleGuide/current/StyleGuide/Inclusivity

Reply:
just a meme, sorry

Reply:
My very first programming book was _Beginning Programming for Dummies_. I was doomed from the start.